python/feathub/feature_views/transforms/join_transform.py

```diff
@@ -25,26 +25,26 @@ class JoinTransform(Transformation):
     def __init__(
         self,
         table_name: str,
-        feature_name: str,
+        expr: str,
```

Comment:
Can you use `@deprecated_alias` to keep supporting `expr` for backward compatibility?